main/strlist.h

```diff
@@ -15,16 +15,15 @@
 #include "general.h"  /* must always come first */
 
 #include "vstring.h"
+#include "ptrarray.h"
 
 #include <stdio.h>
 
 /*
 *   DATA DECLARATIONS
 */
 typedef struct sStringList {
-	unsigned int max;
-	unsigned int count;
-	vString    **list;
+	ptrArray *array;
 } stringList;
```

Comment:
this could be merely `typedef ptrArray stringList`, and would avoid the intermediate allocation.